cdk/lib/monitoring.ts

```diff
@@ -15,14 +17,32 @@ export class Monitoring extends GuStack {
 
 		const lambdaBaseName = 'cmp-monitoring';
 
-		new GuLambdaFunction(this, lambdaBaseName, {
-			app: `${lambdaBaseName}-lambda-${region}`,
-			functionName: `${lambdaBaseName}-${stage}`,
-			fileName: `${lambdaBaseName}-lambda-${region}.zip`,
-			handler: 'index.handler',
-			runtime: Runtime.NODEJS_14_X,
-			timeout: Duration.seconds(300),
-			memorySize: 2048,
+		const monitoringLambdaFunction = new GuLambdaFunction(
+			this,
+			lambdaBaseName,
+			{
+				app: `${lambdaBaseName}-lambda-${region}`,
+				functionName: `${lambdaBaseName}-${stage}`,
+				fileName: `${lambdaBaseName}-lambda-${region}.zip`,
+				handler: 'index.handler',
+				runtime: Runtime.NODEJS_14_X,
+				timeout: Duration.seconds(300),
+				memorySize: 2048,
+			},
+		);
+
+		const lambdaEventTarget = new LambdaFunction(monitoringLambdaFunction, {
+			event: RuleTargetInput.fromObject({
+				stage: 'PROD', // Both scheduled cmp-monitoring-CODE and cmp-monitoring-PROD are monitoring prod versions
+				region: region,
+			}),
+		});
+
+		const monitoringDuration: number = stage === 'PROD' ? 2 : 5; // Every 5 minutes for CODE; Every 2 minutes for PROD.
```

Comment:
I think we probably don't care if this runs at all in CODE really, this could be once a day.

Reply:
Thanks, I've updated this,